packages/fxa-shared/.vscode/tasks.json

```diff
@@ -16,7 +16,7 @@
     {
       "label": "Run Current Test",
       "type": "shell",
-      "command": "./node_modules/mocha/bin/mocha",
+      "command": "./node_modules/mocha/bin/mocha.js",
```

Comment:
A couple more references buried in the bowels of */.vscode/tasks.json files:

```sh
git grep -n "bin/mocha" packages | cat
packages/fxa-auth-server/scripts/test-remote-quick.js:16:    path.join(path.dirname(__dirname), 'node_modules/.bin/mocha.js'),
packages/fxa-event-broker/.vscode/tasks.json:19:      "command": "./node_modules/mocha/bin/mocha",
packages/fxa-graphql-api/.vscode/tasks.json:19:      "command": "./node_modules/mocha/bin/mocha",
packages/fxa-shared/.vscode/tasks.json:19:      "command": "./node_modules/mocha/bin/mocha.js",
packages/fxa-support-panel/.vscode/tasks.json:20:      "command": "./node_modules/mocha/bin/mocha",
```

Reply:
I saw some of these and was uncertain whether to update all of them -- a couple of them don't have mocha as a dependency, so I wasn't sure how they could be accessing it. Do you know which package's copy they could be piggybacking off of?